src/cli/run_command.zig

```diff
@@ -1731,6 +1732,43 @@ pub const RunCommand = struct {
 
         return false;
     }
+
+    pub fn execAsIfNode(ctx: Command.Context) !void {
+        std.debug.assert(cli.pretend_to_be_node);
+
+        if (ctx.positionals.len == 0) {
+            Output.errGeneric("Missing script to execute. Bun's provided 'node' cli wrapper does not support a repl.", .{});
+            Global.exit(1);
+        }
+
+        // TODO(@paperdave): merge windows branch
+        // var win_resolver = resolve_path.PosixToWinNormalizer{};
+
+        const filename = ctx.positionals[0];
+
+        const normalized_filename = if (std.fs.path.isAbsolute(filename))
+            // TODO(@paperdave): merge windows branch
+            // try win_resolver.resolveCWD("/dev/bun/test/etc.js");
+            filename
+        else brk: {
+            const cwd = try bun.getcwd(&path_buf);
```

Comment:
we haven't already loaded the cwd at this point?

Reply:
i dont think so. code copied lightly from the existing code path